sycl/test-e2e/DeviceLib/math_fp64_test.cpp

```diff
@@ -1,6 +1,6 @@
 // REQUIRES: aspect-fp64
 // UNSUPPORTED: hip
-// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %{build} -o %t.out
 // RUN: %CPU_RUN_PLACEHOLDER %t.out
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
 // RUN: %ACC_RUN_PLACEHOLDER %t.out
```

Comment:
Any reason this and the compilation line below are left as is? The same goes for the first batch of run lines in math_test.cpp

Reply:
Because of lines 9-10. I'm working on enabling device-specific `%if` one the line with `%{run}` that would enable changes in this file.